packages/@atjson/offset-annotations/src/annotations/threads-embed.ts

```diff
@@ -0,0 +1,6 @@
+import { IframeEmbed } from "./iframe-embed";
+
+export class ThreadsEmbed extends IframeEmbed {
```

Comment:
To save future refactoring, I would like this not to be an iframe. Could you also link developer documentation in the comments and clearly describe what this block is for? Thanks!

Reply:
Good call. We probably want to support most of the same attributes though, except for height/width. Perhaps you can have it extend
```
BlockAnnotation<{
  url: string;
  /**
   * Refers to a slice instead of being an
   * embedded document.
   */
  caption?: string;
  sandbox?: string;
  /**
   * A named identifier used to quickly jump to this item
   */
  anchorName?: string;
}>
```

Reply:
Please don't use caption here, and instead use `content` with that being the tich text content of the post from inside the blockquote.

Reply:
nb. This block _will_ have to support layout in the future, since that is a universal thing.

Reply:
@tim-evans can you please clarify for Indrani what you mean here. For ex, I know that you want to align this work with your incoming PR (@Indrani-Gostu  see https://github.com/CondeNast/atjson/pull/1685/files#diff-ab0804375eade56aa724d6ba38750b3e8d44187dab2e6b49f356dbf60ad460aeR30). It looks like those attributes are not being written in any converter though - is that expected?